src/parsec_client/wire_protocol.md

```diff
@@ -235,58 +227,33 @@ service back to the client). However, most fields are relevant and common to bot
+| Provider             | Common         | 1               | Identify the back-end service provider for which the request is intended. A value of zero indicates that the request is intended for a special provider, which always exists, and is used for service discovery and communication bootstrapping.                                                                                                                                                                                                    |
+| Session handle       | Common         | 8               | Session identifier.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Content type         | Common         | 1               | Defines how the message body should be processed. The only currently-supported value is `0x01`, which indicates that the message body should be treated as a serialized protobuf message.                                                                                                                                                                                                                                                           |
+| Accept type          | Requests only  | 1               | Defines how the service should provide its response. The only currently-supported value is `0x01`, which indicates that the service should provide a response whose body is a serialized protobuf message.                                                                                                                                                                                                                                          |
+| Auth type            | Requests only  | 1               | Defines how the authentication bytes should be interpreted.                                                                                                                                                                                                                                                                                                                                                                                         |
+| Content length       | Common         | 4               | Provides the exact number of bytes of content.                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I wonder if we should call this `Body length` instead - is the header considered content or not? The paragraphs below refer to it as `message body`. It does say that the length of the body must match `Content Length`, but I find it a bit confusing.